src/app/hooks/useGoogleMapAPI.js

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from "react";
+import loadScript from "../utils/loadScript";
+
+/**
+ * useGoogleMapApi
+ *
+ * @returns {false | true} google map api is loaded
+ */
+export default function useGoogleMapApi() {
+  const [loaded, setLoaded] = useState(false);
+
+  useEffect(() => {
+    // Check if paypal has already been loaded
+    if (document.querySelector("#google-maps")) {
```

Comment:
Would it be useful to pass in this id into the hook?

Reply:
Also the comment lol

Reply:
not unless we decide to make a hook for useScript instead....
let me remove that comment hahahahah

Reply:
I mean, we could do, in the future
useScripts: id, src